stl/inc/execution

```diff
@@ -4009,7 +4009,7 @@ _Ty _Reduce_move_unchecked(_InIt _First, const _InIt _Last, _Ty _Val, _BinOp _Re
         return _Reduce_plus_arithmetic_ranges(_First, _Last, _Val);
     } else {
         for (; _First != _Last; ++_First) {
-            _Val = _Reduce_op(_STD move(_Val), _STD move(*_First)); // Requirement missing from N4713
+            _Val = _Reduce_op(_STD move(_Val), _STD move(*_First)); // Requirement missing from N4950

```

Comment:
A requirement that would allow us to perform this operation. In many cases, the numeric algorithms added in C++17 don't have sufficient constraints to be implementable.